Source/JavaScriptCore/API/JSValueRef.h

```diff
@@ -292,6 +320,133 @@ JS_EXPORT JSValueRef JSValueMakeString(JSContextRef ctx, JSStringRef string);
+    @function
+    @abstract         Truncates a JavaScript BigInt to an unsigned 64-bit integer and returns the resulting integer.
+    @param ctx        The execution context to use.
+    @param value      The BigInt JSValue to truncate.
+    @param exception  A pointer to a JSValueRef in which to store an exception, if any. Pass NULL if you do not care to store an exception.
+    @result           A uint64_t with the result of conversion, or 0 if an exception is thrown.
+    @discussion       If the value is not a BigInt, an exception is thrown.
+*/
+JS_EXPORT uint64_t JSBigIntToUInt64(JSContextRef ctx, JSValueRef value, JSValueRef* exception) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+    @function
+    @abstract         Truncates a JavaScript BigInt to a singed 64-bit integer and returns the resulting integer.
+    @param ctx        The execution context to use.
+    @param value      The BigInt JSValue to truncate.
+    @param exception  A pointer to a JSValueRef in which to store an exception, if any. Pass NULL if you do not care to store an exception.
+    @result           A int64_t with the result of conversion, or 0 if an exception is thrown.
+    @discussion       If the value is not a BigInt, an exception is thrown.
+*/
+JS_EXPORT int64_t JSBigIntToInt64(JSContextRef ctx, JSValueRef value, JSValueRef* exception) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+    @function
+    @abstract         Truncates a JavaScript BigInt to a double and returns the resulting double.
+    @param ctx        The execution context to use.
+    @param value      The BigInt JSValue to truncate.
+    @param exception  A pointer to a JSValueRef in which to store an exception, if any. Pass NULL if you do not care to store an exception.
+    @result           A double with the result of conversion, or 0 if an exception is thrown.
+    @discussion       This is equivalent to calling the `Number` constructor from JavaScript with a BigInt. If the value is not a BigInt, an exception is thrown.
+*/
+JS_EXPORT double JSBigIntToDouble(JSContextRef ctx, JSValueRef value, JSValueRef* exception) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+    @function
+    @abstract         Converts a BigInt JSValue to JSString.
+    @param ctx        The execution context to use.
+    @param value      The BigInt JSValue to convert to string.
+    @param radix      An integer in the range 2 to 36 to be used as the base for representing the BigInt value.
+    @param exception  A pointer to a JSValueRef in which to store an exception, if any. Pass NULL if you do not care to store an exception.
+    @result           A JSString with the result of conversion, or NULL if an exception is thrown. Ownership follows the Create Rule.
+    @discussion       This is equivalent to calling the `BigInt::toString` from JavaScript with a radix. If the value is not a BigInt or the radix is not in range 2 to 36, an exception is thrown.
```

Comment:
Might also be worth mentioning that `JSBigIntToString` with a radix of `10` is the same as calling `JSValueToString`